openaev-api/src/main/java/io/openaev/rest/connector_instance/ConnectorInstanceApi.java

```diff
@@ -0,0 +1,136 @@
+      content =
+          @Content(
+              mediaType = "application/json",
+              array =
+                  @ArraySchema(
+                      schema = @Schema(implementation = ConnectorInstanceConfiguration.class))))
+  public Set<ConnectorInstanceConfiguration> getConnectorInstanceConfiguration(
+      @PathVariable @NotBlank final String connectorInstanceId) {
+    return connectorInstanceService.getConnectorInstanceConfigurations(connectorInstanceId);
+  }
+
+  @PutMapping(value = CONNECTOR_INSTANCE_URI + "/{connectorInstanceId}/configurations")
+  @Operation(summary = "Update connector instance configuration")
+  @RBAC(actionPerformed = Action.WRITE, resourceType = ResourceType.CATALOG)
+  @ApiResponse(
+      responseCode = "200",
+      content =
+          @Content(
+              mediaType = "application/json",
+              array =
+                  @ArraySchema(
+                      schema = @Schema(implementation = ConnectorInstanceConfiguration.class))))
+  public List<ConnectorInstanceConfiguration> updateConnectorInstanceConfigurations(
+      @PathVariable @NotBlank final String connectorInstanceId,
+      @Valid @RequestBody CreateConnectorInstanceInput input) {
+    return orchestrationService.updateConnectorInstanceConfiguration(connectorInstanceId, input);
+  }
+
+  @GetMapping(value = CONNECTOR_INSTANCE_URI + "/{connectorInstanceId}/logs")
+  @Operation(summary = "Retrieve connector instance logs")
+  @RBAC(actionPerformed = Action.READ, resourceType = ResourceType.CATALOG)
+  @ApiResponse(
+      responseCode = "200",
+      content =
+          @Content(
+              mediaType = "application/json",
+              array = @ArraySchema(schema = @Schema(implementation = ConnectorInstanceLog.class))))
+  public List<ConnectorInstanceLog> retrieveConnectorInstanceLogs(
+      @PathVariable @NotBlank final String connectorInstanceId) {
+    return connectorInstanceLogService.findLogsByConnectorInstanceId(connectorInstanceId);
+  }
+
+  @PutMapping(value = CONNECTOR_INSTANCE_URI + "/{connectorInstanceId}/requested-status")
+  @Operation(
+      summary = "Update requested status",
+      description = "Update requested status of connector instance")
+  @RBAC(actionPerformed = Action.WRITE, resourceType = ResourceType.CATALOG)
+  @ApiResponses(
+      value = {
+        @ApiResponse(responseCode = "200", description = "Successfully updated requested status")
+      })
+  public ConnectorInstance updateRequestedStatus(
+      @PathVariable @NotBlank final String connectorInstanceId,
+      @Valid @RequestBody UpdateConnectorInstanceRequestedStatus input) {
+    return orchestrationService.updateRequestedStatus(
+        connectorInstanceId, input.getRequestedStatus());
+  }
+
+  @DeleteMapping(value = CONNECTOR_INSTANCE_URI + "/{connectorInstanceId}")
+  @Operation(summary = "Delete connector instance")
+  @RBAC(actionPerformed = Action.DELETE, resourceType = ResourceType.CATALOG)
```

Comment:
There doesn't seem to be any RBAC on any method allowing the creation of catalog (I couldn't find any @ RBAC(actionPerformed = Action.CREATE, resourceType = ResourceType.CATALOG) . Is that normal ?